### PR TITLE
DEP Update dependency versions

### PR DESCRIPTION
**Pull request by emteknetnz**

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -7,44 +7,44 @@
...
-        "tractorcow/silverstripe-fluent": "4.x-dev"
+        "cwp/starter-theme": "3.2.x-dev",
+        "cwp/agency-extensions": "2.5.x-dev",
+        "cwp/watea-theme": "3.1.x-dev",
+        "cwp/cwp-core": "2.8.x-dev",
+        "cwp/cwp": "2.8.x-dev",
+        "cwp/cwp-pdfexport": "1.2.x-dev",
+        "dnadesign/silverstripe-elemental-userforms": "3.0.x-dev",
+        "symbiote/silverstripe-multivaluefield": "5.1.x-dev",
+        "tractorcow/silverstripe-fluent": "4.1.x-dev"
```
Comment (maxime-rainville) [inline composer.json]:
```suggestion
        "tractorcow/silverstripe-fluent": "4.5.x-dev"
```

**Review by maxime-rainville — CHANGES_REQUESTED** [review]

**Comment by emteknetnz** [comment]
Updated

**Review by maxime-rainville — APPROVED** [review]
Looks good. Let's maybe wait until https://github.com/silverstripe/recipe-form-building/pull/14 and and https://github.com/silverstripe/cwp-recipe-search/pull/19 are merged before merging this one.

**Comment by maxime-rainville** [comment]
Merging despite broken build so we can get the CI running against the correct branches.

**Pull request merged**